PDO/PDO.php

```diff
@@ -1115,7 +1115,12 @@ public function query($statement, $mode = PDO::ATTR_DEFAULT_FETCH_MODE, $arg3 =
      * @see PDOStatement::setFetchMode For a full description of the second and following parameters.
      */
     #[PhpStormStubsElementAvailable('8.0')]
-    public function query($statement, $mode = PDO::ATTR_DEFAULT_FETCH_MODE, ...$fetch_mode_args) {}
+    #[LanguageLevelTypeAware(['8.0' => 'array'], default: 'PDOStatement|false')]
+    public function query(
```

Comment:
Thank you, fixed 👍

Reply:
Hi @romantymoshyk , I see some inconsistency here. Probably because the attribute format isn't described very well in docs so let me please explain it a bit just in case: `#[LanguageLevelTypeAware(['8.0' => 'array'], default: 'PDOStatement|false')]` means that the function supposed to have the return type `array` for all versions **since** 8.0 and for earlier versions return type is supposed to be `PDOStatement|false` but the function has `#[PhpStormStubsElementAvailable('8.0')]` and it means that according to such signature function will have only `array` return type. According to this did you mean `#[LanguageLevelTypeAware(['8.0' => 'PDOStatement|false'], default: 'array')]`?